From: Martin Pitt <martin@piware.de>
Date: Wed, 18 Apr 2007 19:33:07 +0000 (-0400)
Subject: support more Macbook Pro models
X-Git-Tag: HAL_0_2
X-Git-Url: http://gitweb.freedesktop.org/?p=hal.git;a=commitdiff;h=98e53bff6ee6c94113b2e745f727c7a78bae5f06

support more Macbook Pro models

Matthew recently fixed the FDIs to support newer MacBook Pro models.

Patch applies cleanly to 0.5.9 and git head.

(changed by davidz: s/smbios.system.product/system.hardware.product)
---

Index: b/fdi/policy/10osvendor/10-macbookpro-utils.fdi
===================================================================
--- a/fdi/policy/10osvendor/10-macbookpro-utils.fdi	2007-04-30 14:56:39.000000000 -0400
+++ b/fdi/policy/10osvendor/10-macbookpro-utils.fdi	2007-04-30 14:56:58.000000000 -0400
@@ -11,6 +11,21 @@
           <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_keyboard_backlight"/>
           <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_lcd_panel"/>
         </match>
+        <match key="system.hardware.product" string="MacBookPro1,2">
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_light_sensor"/>
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_keyboard_backlight"/>
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_lcd_panel"/>
+        </match>
+        <match key="system.hardware.product" string="MacBookPro2,1">
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_light_sensor"/>
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_keyboard_backlight"/>
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_lcd_panel"/>
+        </match>
+        <match key="system.hardware.product" string="MacBookPro2,2">
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_light_sensor"/>
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_keyboard_backlight"/>
+          <spawn udi="/org/freedesktop/Hal/devices/macbook_pro_lcd_panel"/>
+        </match>
       </match>
     </match>
   </device>
